src/ngMock/angular-mocks.js

```diff
@@ -1744,7 +1744,8 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
    *
    * @param {string|RegExp|function(string)=} url HTTP url or function that receives a url
    *   and returns true if the url matches the current definition.
-   * @param {Object=} headers HTTP headers.
+   * @param {(Object|function(Object))=} headers HTTP headers or function that receives http header
+   *   object and returns true if the headers match the current expectation.
```

Comment:
_expectation_ --> definition_ (?)
(To be consistent with the `url` param description above.)

Reply:
It's [the other way round](https://docs.angularjs.org/api/ngMock/service/$httpBackend#request-expectations-vs-backend-definitions): `expect*` create "expectations", `when*` create "definitions". The description for `url` needs to be changed here.